crates/typst-svg/src/lib.rs

```diff
@@ -452,6 +452,14 @@ impl SVGRenderer {
             Size::new(Abs::pt(width), Abs::pt(height)),
             self.text_paint_transform(state, &text.fill),
         );
+        if let Some(stroke) = &text.stroke {
+            self.write_stroke(
+                stroke,
+                Size::new(Abs::pt(width), Abs::pt(height)),
+                self.text_paint_transform(state, &text.fill),
+            );
+            self.xml.write_attribute("paint-order", "stroke"); // make sure the stroke is rendered on top of the fill
```

Comment:
```suggestion

            // Make sure the stroke is rendered on top of the fill.
            self.xml.write_attribute("paint-order", "stroke"); 
```
Please put comments on their own line above the code they document, start them with a capital letter, and end them with a period.

Reply:
From my reading of MDN `paint-order: stroke` draws the stroke first and then the fill. This means the stroke would be rendered _below_ the fill, wouldn't it?

That would be different behaviour from PDF export, which is always fill-stroke.

Reply:
Yes. That's the problem. Which one should we choose? For fake bold usage, the stroke and the fill has the same color so the order donest really matter. But for use cases in #2474, I think the stroke-fill order should be better.

I think we can use the stroke-fill order, and emulate this in pdf by writing stroked glyphs twice. First time stroke only, and second time fill only.

Reply:
We definitely to do it in a way to is consistent for all outputs. Either we just settle on the fill-stroke order for now or we write twice in PDF. I am fine with both ways.

Reply:
I did some try to write them twice and found it not as easy as I expected. I prefer left it to following prs